server/src/subscriber/subscriber.controller.ts

```diff
@@ -53,6 +60,16 @@ export class SubscriberController {
     response.status(200).send({
       status: "success",
     })
+
+    const errorInfo = {
+      email: request.body.email,
+      anonymous_id: addToQueue.anonymous_id,
+      lists: this.list
+    }
+
+    // Now we keep checking to make sure the import was successful
+    const importConfirmation = await this.subscriberService.checkCreate(addToQueue.result[1]["job_id"], response, 0, CHECKS_BEFORE_FAIL, PAUSE_BETWEEN_CHECKS, this.list, errorInfo);
```

Comment:
Do we need this variable if we're not using it? This could flag as an unused variable in the linter.

Reply:
It's fine, it will be used when we add the subscription confirmation email